s2apler/model.py

```diff
@@ -569,7 +568,7 @@ def predict(
                 max_label = 0
 
             # now we can rejoin the null titles based only on the ids
-            paper_ids_for_nulls = block_dict_no_null_titles.get(block_key, [])
+            paper_ids_for_nulls = block_dict_null_titles.get(block_key, [])
```

Comment:
Is this a bug? What's going on with the changes in model.py?

Reply:
Not a bug, but the variable naming above is confusing or swapped.